### ISSUE TITLE
Anybody tried to install in OP7 pro 5G model ?

### ISSUE THREAD

**Issue by biboyrafik25** [phhusson/treble_experimentations#1254]
i tried several ways ton install it but device stucks in fastboot mod help plz

**Comment by phhusson** [comment]
You flashed vbmeta?

Le dim. 12 avr. 2020 à 11:20, biboyrafik25 <notifications@github.com> a
écrit :

**Comment by biboyrafik25** [comment]
of course

**Comment by phhusson** [comment]
`fastboot getvar all`?

Le dim. 12 avr. 2020 à 11:38, biboyrafik25 <notifications@github.com> a
écrit :

**Comment by biboyrafik25** [comment]
[Sans titre.pdf](https://github.com/phhusson/treble_experimentations/files/4466192/Sans.titre.pdf)

**Comment by phhusson** [comment]
Did you flash vbmeta on the first try, or did you try without it before
that?
Do you remember whether you flashed GSI to slot b or slot a?
Try flashing stock boot.img and reboot

Le dim. 12 avr. 2020 à 13:00, biboyrafik25 <notifications@github.com> a
écrit :

**Comment by biboyrafik25** [comment]
i allready flash it with all test i know its necessary
so installation step by step is ?

**Comment by phhusson** [comment]
installation step by step is flash stock unmodified OEM rom, flash
stock vbmeta with `fastboot --disable-verity --disable-verification
vbmeta vmbeta.img`, flash arm64 ab system.img, do a factory reset

**Comment by biboyrafik25** [comment]
reflashing stock firmware ok.
then 👍 ahmedchekhab@iMac-de-ahmed dossier sans titre % fastboot --disable-verity --disable-verification flash vbmeta vbmeta.img
Rewriting vbmeta struct at offset: 0
Sending 'vbmeta' (8 KB)                            OKAY [  0.010s]
Writing 'vbmeta'                                   OKAY [  0.003s]
Finished. Total time: 0.035s
ahmedchekhab@iMac-de-ahmed dossier sans titre % fastboot flash system system.img
Sending sparse 'system_b' 1/3 (786428 KB)          OKAY [ 27.662s]
Writing 'system_b'                                 OKAY [  0.003s]
Sending sparse 'system_b' 2/3 (786428 KB)          OKAY [ 29.241s]
Writing 'system_b'                                 OKAY [  0.003s]
Sending sparse 'system_b' 3/3 (429208 KB)          OKAY [ 20.990s]
Writing 'system_b'                                 OKAY [  0.003s]
Finished. Total time: 78.046s
ahmedchekhab@iMac-de-ahmed dossier sans titre % fastboot erae userda   
fastboot: usage: unknown command erae
ahmedchekhab@iMac-de-ahmed dossier sans titre % fastboot erase userdata
******** Did you mean to fastboot format this ext4 partition?
Erasing 'userdata'                                 OKAY [  0.273s]
Finished. Total time: 0.283s
ahmedchekhab@iMac-de-ahmed dossier sans titre % fastboot reboot
Rebooting                                          OKAY [  0.002s]
Finished. Total time: 0.002s

stucks in fastboot mod

**Comment by phhusson** [comment]
...
It's normal to be stuck on fastboot if you did `fastboot erase userdata`
Boot in recovery and do factory reset from recovery.

Le dim. 12 avr. 2020 à 15:03, biboyrafik25 <notifications@github.com> a
écrit :

**Comment by biboyrafik25** [comment]
impossible to boot in recovery reboot in fastboot mod

**Comment by phhusson** [comment]
Ok flash again stock OEM boot.img, and do `fastboot format userdata`

Le dim. 12 avr. 2020 à 15:27, biboyrafik25 <notifications@github.com> a
écrit :

**Comment by biboyrafik25** [comment]
Done. boot ok but no network....sim card allready installed 
![Screenshot_20200412-155100](https://user-images.githubusercontent.com/55406730/79070518-d5cc4100-7cd6-11ea-8ac0-c7bb4aee3245.png)
![Screenshot_20200412-155150](https://user-images.githubusercontent.com/55406730/79070523-d82e9b00-7cd6-11ea-8e0a-43734da7d8bd.png)

**Comment by phhusson** [comment]
ok, now reboot into recovery and do factory reset from recovery

Le dim. 12 avr. 2020 à 16:01, biboyrafik25 <notifications@github.com> a
écrit :

**Comment by biboyrafik25** [comment]
its done but same issue

**Comment by phhusson** [comment]
Ok, well then give me the result of `adb logcat -b all -d`

**Comment by biboyrafik25** [comment]
weird my mac "your mac has encountered an error...." and reboot when i tried to flash vendor image
i tried twice and same issue my phone is ok
[logcat.pdf](https://github.com/phhusson/treble_experimentations/files/4466520/logcat.pdf)

**Comment by phhusson** [comment]
That file format is unusable.
`adb logcat -b all -d > log.txt` and attach the generated `log.txt` to this
issue

Le dim. 12 avr. 2020 à 16:59, biboyrafik25 <notifications@github.com> a
écrit :

**Comment by biboyrafik25** [comment]
sorry but i use this command in Terminal (mac) ?

**Comment by phhusson** [comment]
yes

**Comment by biboyrafik25** [comment]
sorry im a noob i know where can i find the log.txt ?
Last login: Sun Apr 12 16:48:04 on ttys000
@iMac-de-ahmed ~ % adb logcat -b all -d > log.txt
@iMac-de-ahmed ~ %

**Comment by phhusson** [comment]
In your shell's `~` which is your unix home directory

Le dim. 12 avr. 2020 à 17:22, biboyrafik25 <notifications@github.com> a
écrit :

**Comment by biboyrafik25** [comment]
thx 
[log.txt](https://github.com/phhusson/treble_experimentations/files/4466572/log.txt)

**Comment by phhusson** [comment]
What's the current value of ro.telephony.default_network ?
Try setting it to 11, 20, 23 24, 26

Le dim. 12 avr. 2020 à 17:33, biboyrafik25 <notifications@github.com> a
écrit :

**Comment by biboyrafik25** [comment]
Soory i dont understand

**Comment by phhusson** [comment]
ok first of all, does wifi work?
then give the result of
`adb shell getprop ro.telephony.default_network`

**Comment by biboyrafik25** [comment]
wifi works
file is empty..... its possible ?

**Comment by phhusson** [comment]
yes it's possible it's empty, and that's actually what I expected.
Okay, now do this:
```
> adb shell
$ su
# setprop ro.telephony.default_network 23
# getprop ro.telephony.default_network <<<< this is to check the setprop
worked fine
# stop;start
```

Here android should soft-reboot. Does SIM appear after that?

Le dim. 12 avr. 2020 à 18:07, biboyrafik25 <notifications@github.com> a
écrit :

**Comment by biboyrafik25** [comment]
1|:/data/data # getprop ro.telephony.default_network                           
23
after reboot sim not appear :(

**Comment by biboyrafik25** [comment]
Hello, i dont understand this : my model is gm1920 (model 5G) not gm1917 (model 4G). Thx
![79070518-d5cc4100-7cd6-11ea-8ac0-c7bb4aee3245](https://user-images.githubusercontent.com/55406730/79106554-0a8dd600-7d73-11ea-8bbb-5acff0086aa9.png)

**Comment by biboyrafik25** [comment]
Bah issue not solved 5g model not a good choice return device to seller and changing with realme x2 pro
